### PR TITLE
ensure proper request content type

### PR DESCRIPTION
**Pull request by vibrant**
Requests to Vero were failing because of a wrong request content type.  Please merge and release an updated version on PyPi.

### CODE REVIEW

File: vero/client.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class VeroEventLogger(object):
     @classmethod
     def _fire_request(cls, endpoint, payload):
         json_payload = json.dumps(payload)
-        return requests.request(endpoint.method, endpoint.url, data=json_payload)
+        return requests.request(endpoint.method, endpoint.url, data=json_payload, headers={'Content-Type': 'application/json'})
```
Comment (dlanger) [inline vero/client.py]: Is the `json` parameter of `requests` supported here? See [this](http://stackoverflow.com/a/26344315) for an example.

**Comment by vibrant** [comment]
Didn't know about that one, so I guess it's good if you just switch data= to json= and remove the dumping?

**Comment by ryanwilsonperkin** [comment]
That's correct @vibrant, can you update this PR to use the `json` keyword argument?

**Comment by vibrant** [comment]
Done

**Comment by ryanwilsonperkin** [comment]
Thanks @vibrant, I've checked out your branch locally for testing and linting and it appears there are two things left before we can merge this.

1. There's a failing test here: https://github.com/waveaccounting/vero-python/blob/master/vero/client.py#L28 that checks that requests was called with the proper `data` parameter. You can update this to be the `json` parameter and remove the use of `json.dumps`.
2. We no longer need to import the `json` module in either the __client.py__ or __tests/client_test.py__ (after the first point has been addressed).

**Comment by ryanwilsonperkin** [comment]
On second thought, I'll merge this PR as is and add the tests/lint-fix in a separate PR.
Thanks for your contribution @vibrant !

**Comment by ryanwilsonperkin** [comment]
This has now been released as v2.0.0 and is available on PyPi

**Pull request merged**